R/run_cluster_analysis.R

```diff
@@ -1,13 +1,18 @@
-#' Load data onto the server associated with
-#' a BiodiverseR::basedata object
+#' Runs a cluster analysis with the given parameters
+#' Uses the given basedata object to call the server
 #'
 #'
 #' @param bd list
-#' @param r_data list
-#' @param raster_params list
-#' @param spreadsheet_params list
-#' @param delimited_text_params list
-#' @param shapefile_params list
+#' @param index character
+#' @param linkage_function character
+#' @param calculations character
+#' @param spatial_conditions character
+#' @param def_query character
+#' @param name character
+#' @param tree list
+#' @param cluster_tie_breakers
```

Comment:
Type hint?  Or has not been implemented in the code yet?

Reply:
i couldn't find any implementation of cluster_tie_breaker

Reply:
Looks like we need to add it then.  But not in this PR.